routes/message.routes.js

```diff
@@ -16,7 +17,10 @@ router.post("/messages", (req, res) => {
         "The keys can't be empty. The necessary keys are destination and message",
     });
   }
-  if (destination && typeof destination !== "string" || body && typeof body !== "string") {
+  if (
+    (destination && typeof destination !== "string") ||
```

Comment:
be careful with have you tried to see what will happen if you the value is null?

what will happen if you have (null && typeof null !== "string") what will be the result of this expression?